tests/yam/yast_container/yast2_control_center.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Start containerized YaST Control Center with ncurses
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'basetest';
+use strict;
+use warnings;
+use testapi;
+
+my $podman_cmd = "podman container runlabel run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses-test-api";
```

Comment:
should be inside the `run` otherwise I think it is global variable, we probably want all together, so it is easier to refactor.